### PR TITLE
Repo avatar fixes

### PR DESCRIPTION
**Pull request by silverwind**
- Split up avatar rendering helpers for performance
- Fix showing repo SVG icon when no avatar is set
- Make repo SVG and avatar same size at 32px
- Fix fork line by adding vertical flexbox on repo title

Screen Shot 2020-12-08 at 01 14 15
Screen Shot 2020-12-08 at 01 14 23
Screen Shot 2020-12-07 at 21 42 04
Screen Shot 2020-12-07 at 21 41 53

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -651,6 +651,7 @@ $(FOMANTIC_DEST): $(FOMANTIC_CONFIGS) | node_modules
 	cp -f web_src/fomantic/theme.config.less node_modules/fomantic-ui/src/theme.config
 	cp -rf web_src/fomantic/_site/* node_modules/fomantic-ui/src/_site/
 	npx gulp -f node_modules/fomantic-ui/gulpfile.js build
+	perl -p -i -e 's#.*to-be-removed.*\n##g' web_src/fomantic/build/semantic.css
```
Comment (silverwind) [inline Makefile]: There's now way around this if we want avatar images to render in the HTML-defined size besides setting invalid values for fomantic's variables which I think is even worse than removing them (which this here does).

Comment (lunny) [reply Makefile]: Could we use other tools rather than a new dependency?

Comment (silverwind) [reply Makefile]: Hmm could also use `sed` I guess which we already depend on.

Comment (silverwind) [reply Makefile]: Thought `sed` can not remove newlines and fomantic build depends on node, so I'll make it a node script which will also be more flexible.

Comment (silverwind) [reply Makefile]: Script added.

Comment (silverwind) [reply Makefile]: Actually, removed again. I think it's not yet warranted to do this kind of processing for something that's not really an issue to override.

**Comment by 6543** [comment]
private mirrors missing "private" label

**Review by 6543 — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Comment by techknowlogick** [comment]
🚀

**Pull request merged**